house/views.py

```diff
@@ -13,11 +15,42 @@ def global_page(request):
 
 
 def house_login(request):
-    raise NotImplementedError
+
+    errorMsg = ""
+
+    # if this is a POST request we need to process the form data
+    if request.method == 'POST':
+
+        try:
+            # create a form instance and populate it with data from the request:
+            form = HouseIDForm(request.POST)
+
+            house_id = form.data["house_id"]
+
+            # if House.objects.filter(house_id=house_id).exists():
+            if House.objects.filter(house_id=house_id).count() == 1:
+
+                return HttpResponseRedirect(f'/../{house_id}')
+
+            else:  # count() == 0
+
+                errorMsg = "There is no House with the provided ID"
+                form = HouseIDForm()
+
+        # In case an exception not a valid UUID is thrown
+        except:
+            errorMsg = "There is no House with the provided ID : " + str(house_id)
+            form = HouseIDForm()
+
+    # if a GET (or any other method) we'll create a blank form
+    else:
+        form = HouseIDForm()
+
+    return render(request, LOGIN_PAGE_ROUTE, {'form': form, 'msg': errorMsg})
 
 
 def house_view(request, house_id):
-    raise NotImplementedError
+    return HttpResponse(f'house_id: {house_id}')
```

Comment:
changed house_view function in order to check that the ID goes to the url as ot needs to

Reply:
I think you can remove this code. I guess you did that only to check that your code works, Am I right?

Reply:
Yep
I'll remove it